src/contents/posts/2023/05/18-push/contents.md

```diff
@@ -0,0 +1,331 @@
+---
+slug: push-notifications-with-swift-server-side
+title: Push notifications with Swift server side in 2023
+description: Looking around options for Push notifications with Swift server side.
```

Comment:
Learn how to send push notifications with Swift on the server side using the Google FCM API and the APN Swift library.